src/main/drivers/timer_def.h

```diff
@@ -701,3 +702,127 @@
 #define DEF_TIM_AF__PI7__TCH_TIM8_CH3     D(3, 8)
 
 #endif
+
+
+#ifdef USE_TIMER_MGMT
+
+#define TIMER_TAG_INDEX(t)           (((t) >> 3) - 1)
+#define TIMER_TAG_NO(t)              ((t) >> 3)
+#define TIMER_TAG_CHANNEL_INDEX(t)   (((t) & 0x7) - 1)
```

Comment:
I would prefer to encode (zero based) channel index in tag, even when it complicates timers without channel ...

Reply:
No I prefer to match the ST channel names. Either way the timerTag_T should never be pulled apart directly hence the helper defines.